datahub/dataset/company_export_to_country/test/test_views.py

```diff
@@ -1,39 +1,33 @@
-import factory
 import pytest
 from django.urls import reverse
 from rest_framework import status
 
-from datahub.company.models import Company
-from datahub.company.test.factories import CompanyFactory
-from datahub.core import constants
+from datahub.company.models import CompanyExportCountry
+from datahub.company.test.factories import CompanyExportCountryFactory, CompanyFactory
 from datahub.core.test.factories import to_many_field
 from datahub.dataset.core.test import BaseDatasetViewTest
-from datahub.metadata.models import Country
-
-
-class CompanyExportToCountriesFactory(factory.django.DjangoModelFactory):
-    """Returns a company-export_to_country relationship"""
-
-    company = factory.SubFactory(CompanyFactory)
-    country_id = constants.Country.united_kingdom.value.id
-
-    class Meta:
-        model = Company.export_to_countries.through
 
 
 class CompanyWithExportToCountriesFactory(CompanyFactory):
     """Extends Company Factory to provide a export_to_countries"""
 
     @to_many_field
-    def export_to_countries(self):
-        """Return stubbed countries"""
-        return list(Country.objects.all()[:1])
+    def export_countries(self):
+        """Return stubbed export countries"""
+        return [
+            CompanyExportCountryFactory(
+                status=CompanyExportCountry.Status.CURRENTLY_EXPORTING,
+            ),
+        ]
 
 
 def get_expected_data_from_company(company):
-    """Returns company export_to_countries data as a list of dictionaries"""
-    values = company.export_to_countries.through.objects.filter(
-        company_id=company.id,
+    """
+    Returns company export_countries data with status `currently_exporting`
+    as a list of dictionaries
+    """
+    values = company.export_countries.all().filter(
```

Comment:
Could get rid of the `.all()` here:
```suggestion
    values = company.export_countries.filter(
```